### PR TITLE
added circle config

### PR DESCRIPTION
**Pull request by whitfield-mj**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+jobs:
+  build:
+    executor:
+      name: hmpps/node
+      tag: << pipeline.parameters.node-version >>
+    steps:
+      - checkout
+      - run:
+          name: Update npm
+          command: 'sudo npm install -g npm@9.2.0'
```
Comment (davidmiddletonmoj) [inline .circleci/config.yml:29]: Does this specifically require 9.2.0 ? We use v10 in other repos, and normally set this to `latest`

**Review by davidmiddletonmoj — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (whitfield-mj) [reply .circleci/config.yml:29]: Yeah missed that on my copy/paste/scan through for updates routine

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,188 @@
+version: 2.1
+
+orbs:
+  hmpps: ministryofjustice/hmpps@8
```
Comment (brightonsbox) [inline .circleci/config.yml:4]: This can be hmpps@9 now

**Review by davidmiddletonmoj — DISMISSED** [review]

**Review by brightonsbox — DISMISSED** [review]
LGMT

**Review by brightonsbox — APPROVED** [review]
LGTM

**Pull request merged**